conf/pacific/cephadm/sanity-cephadm.yaml

```diff
@@ -1,51 +1,51 @@
 globals:
   - ceph-cluster:
-     name: ceph
-     node1:
-       role:
+      name: ceph
+      node1:
+        role:
           - mon
           - mgr
           - installer
           - node-exporter
           - alertmanager
           - grafana
           - prometheus
-     node2:
-       role:
+      node2:
+        role:
           - mon
           - mds
           - node-exporter
           - alertmanager
-     node3:
-       role:
+      node3:
+        role:
           - osd
           - node-exporter
-       no-of-volumes: 4
-       disk-size: 15
-     node4:
-       role:
+        no-of-volumes: 4
+        disk-size: 15
+      node4:
+        role:
           - osd
           - node-exporter
-       no-of-volumes: 4
-       disk-size: 15
-     node5:
-       role:
-         - osd
-         - node-exporter
-       no-of-volumes: 4
-       disk-size: 15
-     node6:
-       role:
+        no-of-volumes: 4
+        disk-size: 15
+      node5:
+        role:
+          - osd
+          - node-exporter
+        no-of-volumes: 4
+        disk-size: 15
+      node6:
+        role:
           - mon
           - rgw
```

Comment:
As there are two `rgw` nodes, do you suggest we try load balancing or HAproxy?

Reply:
Would consider this in next cephadm commits when service/daemons operations are handled.